src/commitment_scheme/kzg10/key.rs

```diff
@@ -325,7 +329,7 @@ impl CommitKey {
 impl OpeningKey {
     /// Checks that a polynomial `p` was evaluated at a point `z` and returned the value specified `v`.
     /// ie. v = p(z).
-    pub fn check(&self, point: Scalar, proof: Proof) -> bool {
+    pub(crate) fn check(&self, point: Scalar, proof: Proof) -> bool {
```

Comment:
`check` is a part of the public API, so `pub` visibility was correct

Reply:
kk